azure-pipelines.yml

```diff
@@ -79,7 +79,7 @@ jobs:
   - bash: ci/scripts/include-guard.sh $SYSTEM_PULLREQUEST_TARGETBRANCH
     condition: eq(variables['Build.Reason'], 'PullRequest')
     displayName: Check formatting on header guards
-  - bash: ci/scripts/verible-format.sh
+  - bash: ci/scripts/verible-format.sh --allowlist
```

Comment:
The fix probably needs to go into the script itself (it's a CI wrapper of the same name to increase confusion)

Reply:
Lol, I should have looked at the path to the script...

Reply:
> it's a CI wrapper of the same name to increase confusion

Sorry!